### PR TITLE
chore: add migration notes from contractkit

### PR DESCRIPTION
**Pull request by nicolasbrugneaux**

### CODE REVIEW

File: packages/sdk/contractkit/MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+I'll show the most "basic" interaction, which is a CELO transfer:
+
+With ethers:
+
+```diff
++ import { tokenAbi } from './abi.json'
+- const CELO = await kit.contracts.getGoldToken();
+- const txReceipt = await CELO.transfer('0x...', amount)
++ const tokenAddress = '0x...'
```
Comment (aaronmgdr) [inline packages/sdk/contractkit/MIGRATION.md]: I think we should show how to get the address from the registry contract here.

File: packages/sdk/contractkit/MIGRATION.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
++ const contract = new ethers.Contract(tokenAddress, tokenAbi, signer);
++ const txReceipt = await contract.transfer('0x...', amount);
```
Comment (aaronmgdr) [inline packages/sdk/contractkit/MIGRATION.md]: this needs to be passed to sendTransaction

**Comment by aaronmgdr** [comment]
There is a lot more ContractKit does than this migration guide covers.

**Review by aaronmgdr — APPROVED** [review]

**Comment by arthurgousset** [comment]
Brief note: 
- [x] publish to celo.academy 
- [ ] share with Viral/Harpal in DevRel (via DevRel channel)

**Comment by nicolasbrugneaux** [comment]
For historical purpose: https://celo.academy/t/migrating-to-ethers-or-viem/3070

**Pull request merged**